io.catenax.return_request/2.0.0/ReturnRequest.ttl

```diff
@@ -0,0 +1,244 @@
+   samm:description "Depicts a time period by specifying start and end date"@en ;
+   samm:dataType :TimePeriodEntity .
+
+:IdentificationCharacteristic a samm:Characteristic ;
+   samm:preferredName "identification characteristic"@en ;
+   samm:description "The unique identifier of the product (via manufacturingPartID, OeNumber and/or serialNumber)\n"@en ;
+   samm:dataType :IdentificationEntity .
+
+:ProductConditionsEntity a samm:Entity ;
+   samm:preferredName "product conditions entity"@en ;
+   samm:description "Entity encapsulating the conditions the product is in."@en ;
+   samm:properties ( [ samm:property :missingParts; samm:optional true ] [ samm:property :dismantled; samm:optional true ] [ samm:property :mechanicalDamage; samm:optional true ] [ samm:property :corroded; samm:optional true ] [ samm:property :discolored; samm:optional true ] :mileage ) .
+
+:TimePeriodEntity a samm:Entity ;
+   samm:preferredName "time period"@en ;
+   samm:description "Entity defining a range of dates to define a time period."@en ;
+   samm:properties ( :startDate :endDate ) .
+
+:IdentificationEntity a samm:Entity ;
+   samm:preferredName "identification entity"@en ;
+   samm:description "Entity encapsulating different unique identifier of the product (manufacturingPartID, OeNumber and/or serialNumber)"@en ;
+   samm:properties ( :manufacturingPartID :oeNumber :serialNumber ) .
+
+:missingParts a samm:Property ;
+   samm:preferredName "missing parts"@en ;
+   samm:description "Completeness of the product, e.g. missing parts are not acceptable."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:dismantled a samm:Property ;
+   samm:preferredName "dismantled"@en ;
+   samm:description "Information on whether the product has been dismantled."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mechanicalDamage a samm:Property ;
+   samm:preferredName "mechanical damage"@en ;
+   samm:description "Information on mechanical damage to the part, e.g. two broken ribs or part deformation indicate mechanical damage."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:corroded a samm:Property ;
+   samm:preferredName "corroded"@en ;
+   samm:description "Information on whether the product has corrosion."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:discolored a samm:Property ;
+   samm:preferredName "discolored"@en ;
+   samm:description "Information on whether the product has been discoloured."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage"@en ;
+   samm:description "Current mileage of the component"@en ;
+   samm:characteristic :Mileage ;
+   samm:exampleValue 89000 .
+
+:startDate a samm:Property ;
+   samm:preferredName "start date"@en ;
+   samm:description "The first day of the period"@en ;
+   samm:characteristic samm-c:Timestamp ;
```

Comment:
the value for characteristic should be `DateTimestamp` here instead of `samm-c:Timestamp`, else the example value is not correct.

Reply:
There is no DateTimestamp in Characteristic class of Aspect Model Editor, which one should I use?
"2022-02-03T14:48:54.709Z" Is this a correct example value? If not, what kind of is correct?